### PR TITLE
Flutter Plugin Upgrade and Build Error solved

### PR DESCRIPTION
**Pull request by nayakastha**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [X] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [X] Check the commit's or even all commits' message styles matches our requested structure.
- [X] Check your code additions will fail neither code linting checks nor unit test.
- [X] I have added necessary documentation (if appropriate)

### Description
- Migrated to AndroidX to keep all the plugins updated
- Changed the Kotlin version for the application to make sure it works for all the latest plugins
- Changed the `flutter_webview_plugin` to `webview_flutter` since the former isn't a maintained plugin 

❤️Thank you!

### CODE REVIEW

File: scp/lib/ui/views/mentor_search/mentor_page.dart [hunk 1]
```diff
@@ -343,17 +344,12 @@ class MentorDetails extends StatelessWidget {
...
-
-      print('add contact dialog closed.');
+  void saveContact(Contact contact) async {
+    // save contact
+    final result = FlutterContacts.insertContact(contact).then((value) {
+      print('add contact dialog closed');
     }).catchError((error) {
-      // FlutterError, most likely unsupported operating system.
-      print('Error adding contact!');
+      print('pata nai bhai kya error hai');
```
Comment (SriramPatibanda) [inline scp/lib/ui/views/mentor_search/mentor_page.dart:352]: If the plugin is creating the issue, then try using [contacts_service](https://pub.dev/packages/contacts_service) plugin which has more likes and popularity.

**Review by SriramPatibanda — CHANGES_REQUESTED** [review]

File: scp/lib/chat.dart [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-        withZoom: true,
-        hidden: true,
-        withJavascript: true,
-        resizeToAvoidBottomInset: true,
-        initialChild: Center(
-          child: CircularProgressIndicator(),
-        ),
-      ),
-    );
+        child: Scaffold(
```
Comment (abhibhaw) [inline scp/lib/chat.dart:10]: Should have used reverse :D

**Review by abhibhaw — APPROVED** [review]
lgtm

File: scp/lib/ui/views/mentor_search/mentor_page.dart (hunk 1 above)
Comment (nayakastha) [reply scp/lib/ui/views/mentor_search/mentor_page.dart:352]: Not an updated plugin. Has the old android embedding.

**Review by dsc-nitr — APPROVED** [review]

**Pull request merged**